front/public/locales/en/operationalStudies/scenario.json

```diff
@@ -53,8 +53,8 @@
     "filterLabel": "Name, label",
     "importTrainSchedule": "Import",
     "invalid": {
-      "pathfinding_not_found": "Pathfinding not found",
-      "pathfinding_failed": "Pathfinding failed",
+      "not_found": "Pathfinding not found",
+      "failed": "Pathfinding failed",
```

Comment:
Why did you rename them ? Since we have `simulation_failed` and `rolling_stock_not_found` below, I think it's more readable to keep the `pathfinding` prefix